protocols/cell-culture.md

```diff
@@ -3,8 +3,8 @@ Plate and grow eukaryotic cells for transfection.
 
 **Plate from frozen state**
 1. Add 10mL DMEM growth medium (+ growth factors) to tissue culture plate.
-2. Move cells from -80C to 37C water bath.
-3. Add 1mL cells to medium, slowly. Gently shake plate to cover. Incubate in 37C. Split about 3 days.
+2. Move cells from -80C to 37C water bath. Centrifuge cells (ask Kim for specific speed and time, but 1500rpm for 5 min should be fine), remove supernatant, and resuspend in 1mL pre-warmed DMEM.

```

Comment:
Question: did 1500 rpm for 5 min. work out fine? If so, no need to optimize further; update the protocol and let's move on.